README.md

```diff
@@ -59,6 +59,7 @@ APIs.
 - [Build](#build)
   - [Linux and macOS](#linux-and-macos)
   - [Windows](#windows-1)
```

Comment:
These two links don't work. Out of scope but calling it out.

Reply:
Fixed.